src/plugins/GUITestBase/src/harness/GUITestLauncher.cpp

```diff
@@ -366,13 +365,19 @@ QString GUITestLauncher::runTest(const QString &testName, const int timeout) {
 QString GUITestLauncher::runTestOnce(U2OpStatus &os, const QString &testName, int iteration, const int timeout, bool enableVideoRecording) {
     QProcessEnvironment environment = prepareTestRunEnvironment(testName, iteration);
 
-    QString path = QCoreApplication::applicationFilePath();
+    QString ugeneUiPath = QCoreApplication::applicationFilePath();
     QStringList arguments = getTestProcessArguments(testName);
 
     // ~QProcess is killing the process, will not return until the process is terminated.
     QProcess process;
     process.setProcessEnvironment(environment);
-    process.start(path, arguments);
+    QString ugeneTestRunWrapper = qgetenv("UGENE_TEST_RUN_WRAPPER");
```

Comment:
I think, all these env variables should be moved to WIKI. It's very difficult to search them every time in the code and a, actually, I don't even remember about at least a half of the existing variables

Reply:
I would put all these options into a separate C++ header file with a detailed documentation.